app_dart/lib/src/request_handlers/scheduler/batch_backfiller_v2.dart

```diff
@@ -7,7 +7,6 @@ import 'package:cocoon_service/src/model/appengine/task.dart';
 import 'package:cocoon_service/src/model/firestore/task.dart' as firestore;
 import 'package:cocoon_service/src/service/datastore.dart';
 import 'package:cocoon_service/src/service/scheduler/policy.dart';
-import 'package:cocoon_service/src/service/scheduler_v2.dart';
```

Comment:
Can this library and `class BatchBackfillerV2` be renamed to exclude the v2?

Reply:
This is addressed at the end of the PR description:

> Even where the v1 versions of classes are removed, the v2 class and file naming is left for now. Cleaning that will be done in a later follow-up since diffs will then correctly show it as a move, whereas if I moved them over the deleted code in the same PR, it would look like this PR was deleting the v2 version and doing major rewrites to v1, making the PR much harder to reason about.

Reply:
sorry, missed that